content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -78,7 +78,8 @@ the policies to enforce STRICT mutual TLS between the workloads.
 
     {{< text bash >}}
     $ kubectl get destinationrule --all-namespaces
-    No resources found.
+        NAMESPACE      NAME                      HOST                                    AGE
+        istio-system   meshexpansion-dr-istiod   istiod.istio-system.svc.cluster.local   5m4s
```

Comment:
Changing this in the doc is not acceptable, since the DR has nothing to do with this task, and is inconsistent with the text above that says make sure there are no existing DRs. I think this is only there if the cluster is configured for multicluster or VM integration. I don't think our test cluster for these tasks should be, but if they will be, we should just change the test.sh to handle this, not the .md (unless we want to update the doc to explain better).

Reply:
Well, the cluster isn't configured for multicluster or VM, unless the test case is doing it.

Will need to see if it's always there.

Reply:
I wonder if the test framework changing to allow multicluster is what's causing this. Maybe need to move from default to none and do the istioctl install would be appropriate.

Reply:
Removed, removed verify. TODO: added to verify ways around having the extra DR in the test (when the docs say one won't be there).